src/main/java/com/maplexpbar/MapleXPBarConfig.java

```diff
@@ -32,14 +32,30 @@ default Skill skill()
 
 	@ConfigItem(
 			position = 2,
+			keyName = "showMaxedSkills",
+			name = "Show Maxed Skills",
+			description = "Display bar even if the skill XP exceeds threshold"
+	)
+	default boolean showMaxedSkills() { return false; }
+
+	@ConfigItem(
+			position = 3,
+			keyName = "maxedThreshold",
+			name = "Maxed XP Threshold",
+			description = "If Show Maxed Skills is checked, use this threshold for XP for considering a skill 'maxed'"
+	)
+	default int maxedThreshold() { return 13034431; }
```

Comment:
Not in love with this being an integer input since the up/down arrows are basically useless at this scale, but it was quick and easy. I thought about an enum with levels 99 and 200mil xp since those are probably the only values someone might use here, but I figured most would leave this alone anyhow, and if someone wants 200mil, that's an easy enough edit? 🤷 open to other ideas here